Formula/rclone.rb

```diff
@@ -0,0 +1,17 @@
+class Rclone < Formula
+  desc "Sync files to and from Google Drive, Amazon Cloud Drive, etc."
+  homepage "http://rclone.org"
+  url "https://github.com/ncw/rclone/releases/download/v1.29/rclone-v1.29-osx-amd64.zip"
+  version "1.29"
+  sha256 "64c95564ae69f5238000c6adb2f4d178923176ecfd9d64cddfd80bfa538e1c15"
+  head "https://github.com/ncw/rclone.git"
+
+  def install
+    bin.install "rclone"
+    man1.install Dir["*.1"]
+  end
+
+  test do
+    system "#{bin}/rclone", "--help"
```

Comment:
At a minimum, this will need to be something like:

```ruby
assert_match "<some suitable part of the output>", shell_output("#{bin}/rclone --help", 1)
```

But we'd really like to have a better test that checks some of the functionality of this software (not sure how feasible this is given that most of it probably needs to interact with some remote service). [Here's a bit more information from our Formula Cookbook.](https://github.com/Homebrew/brew/blob/master/share/doc/homebrew/Formula-Cookbook.md#add-a-test-to-the-formula)